sensu-plugins-elasticsearch.gemspec

```diff
@@ -34,7 +34,7 @@ Gem::Specification.new do |s|
   s.version                = SensuPluginsElasticsearch::Version::VER_STRING
 
   s.add_runtime_dependency 'rest-client',       '1.8.0'
-  s.add_runtime_dependency 'elasticsearch',     '~> 1.0.14'
+  s.add_runtime_dependency 'elasticsearch',     '>= 1.0.10'
```

Comment:
I have opened https://github.com/cdunn/aws-es-transport/pull/4 to allow us to specify greater than es 1.x after this merges and releases we can bump it here...